static/js/admin.js

```diff
@@ -0,0 +1,61 @@
+function popluateTable(bobs) {
+  let $bobTable = $("#bobTable");
+  $bobTable.empty(); // Clear the table before adding to it
+
+  let tableColumns = ["Flavor", "ID", "StartDate", "EndDate", "Active", "Edit", "Delete"];
+
+  let $html = $('<tr>', {});
+  for (let i in tableColumns){
+    $html.append($('<th>', { text: tableColumns[i] }));
+  }
+  $bobTable.append($html);
+  for (var i = 0; i < bobs.length; i++) {
+    $bobTable.append(createBobElement(bobs[i]));
+  }
+}
+
+function createBobElement(bob) {
+  let editButton   = '<a href=./editbob?bobid='+ bob._id + '>Edit Bob</a>';
```

Comment:
you can use jQuery function ".attr( attributeName, value )" instead for consistency & readability